src/lightning_app/utilities/cluster_logs.py

```diff
@@ -43,59 +46,95 @@ def _push_log_events_to_read_queue_callback(read_queue: queue.PriorityQueue):
     Returns callback function used with `on_message_callback` of websocket.WebSocketApp.
     """
 
-    def callback(ws_app: WebSocketApp, msg: str):
-        # We strongly trust that the contract on API will hold atm :D
-        event_dict = json.loads(msg)
-        labels = _ClusterLogEventLabels(**event_dict["labels"])
-
-        if "message" in event_dict:
-            message = event_dict["message"]
-            timestamp = dateutil.parser.isoparse(event_dict["timestamp"])
-            event = _ClusterLogEvent(
-                message=message,
-                timestamp=timestamp,
-                labels=labels,
-            )
-            read_queue.put(event)
+    def callback(_: WebSocketApp, msg: str):
```

Comment:
This isn't the same code, you are returning log_events and it doesn't filter the private logs.

Reply:
I'm not following; this is the same code just refactored?

Reply:
Oh, my bad, I thought this was the app_logs callback. I guess we don't have private logs for our BYOC clusters.